### PR TITLE
Create chemmedchem.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/93085/request-for-a-new-style-in-zotero-for-chemmedchem-journal#latest

### CODE REVIEW

**Comment by adam3smith** [comment]
> use the Angewandte Chemie output style, which is shared by Angewandte Chemie, Chemistry Europe, and ACES journals; 

Didn't check guidelines in detail, but couldn't we make this dependent?

File: dependent/chemmedchem.csl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>ChemMedChem</title>
+    <id>http://www.zotero.org/styles/chemmedchem</id>
+    <link href="http://www.zotero.org/styles/chemmedchem" rel="self"/>
+    <link href="http://www.zotero.org/styles/angewandte-chemie" rel="independent-parent"/>
+    <link href="https://chemistry-europe.onlinelibrary.wiley.com/hub/journal/18607187/notice-to-authors" rel="documentation"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
```
Comment (adam3smith) [inline dependent/chemmedchem.csl:9]: Is the springer documentation linked on purpose?

**Review by adam3smith — COMMENTED** [review]
Quick question

File: dependent/chemmedchem.csl (hunk 1 above)
Comment (POBrien333) [reply dependent/chemmedchem.csl]: Mit dem hin und her wegen der Validierung hab ich den dann am Schluss vergessen. Danke.

**Pull request merged**